core/ui/card_ui_only_qam/only_qam_main.gd

```diff
@@ -35,6 +37,12 @@ func _init():
 
 ## Starts the --only-qam/--qam-only session.
 func _ready() -> void:
+	# Workaround old versions that don't pass launch args via update pack
+	# TODO: Parse the parent PID's CLI args and use those instead.
+	if "--skip-update-pack" in cmdargs and args.size() == 0:
+		logger.warn("Launched via update pack without arguments! Falling back to default.")
+		args = ["steam", "-gamepadui", "-steamos3", "-steampal", "-steamdeck"]
+
```

Comment:
Running only-qam mode was never tested via update pack before ChimeraOS v43 was released. If there is an update ZIP in the user directory, the CLI args gets swallowed and never passed to the child process. This change provides a hacky way to try and set fallback launch arguments.